### DISCUSSION TITLE
SI units: slope vs. rate

### DISCUSSION THREAD

**Discussion by tobolar** [modelica/ModelicaStandardLibrary#4236] [General]
There are defined units indicating change of the unit over time. They have a suffix "slope" or "rate". 

The question is: 
1. Shouldn't this be unified?
2. Shall such units even be a part of Units.SI at all? See https://github.com/modelica/ModelicaStandardLibrary/pull/4103/files#r1152856158

---

Slope:
- Modelica.Units.SI.CurrentSlope
- Modelica.Units.SI.TemperatureSlope
- Modelica.Units.SI.VoltageSlope

Rate:
- Modelica.Units.SI.MassFlowRate
- Modelica.Units.SI.VolumeFlowRate
- Modelica.Units.SI.EntropyFlowRate
- etc.

**Comment by HansOlsson** [comment]
To me the first answer is 'No', and they are different even if the units are similar:
- Slope indicates the derivative of a curve (w.r.t. time).
- FlowRate indicates the rate at which the quantity flows past something (per unit of time). Thus a MassFlowRate doesn't necessarily indicate that the mass changes somewhere (especially not in a closed system), just that it flows past something (a valve etc).

With that line of thinking Current (not CurrentSlope) is a ChargeFlowRate.
So derivatives have slopes and connectors may contain flow-rates.
